.github/PULL_REQUEST_TEMPLATES/add_update_component.md

```diff
@@ -0,0 +1,22 @@
+## 🎯 Changes
+
+<!-- Describe the changes made in this PR. Is it a new component, tool, or template? Or is it an update to an existing one? -->
+
+## 📝 Description
+
+<!-- Provide a detailed description of the changes. What problem does it solve? How does it improve the existing functionality? -->
+
+## 📸 Screenshots (if applicable)
+
+<!-- Include screenshots of the new or updated component/tool/template if applicable. -->
```

Comment:
I don't know about this section.
We don't have a screenshot on packages or templates.